Draft-Accepted/RFCNNNN-PathUpdateForPackageInstall.md

```diff
@@ -0,0 +1,189 @@
+---
+RFC: RFC<four digit unique incrementing number assigned by Committee, this shall be left blank by the author>
+Author: Dongbo Wang
+Status: Draft
+SupercededBy:
+Version: 1.0
+Area: Core
+Comments Due: 5/30/2025
+Plan to implement: Yes
+---
+
+# Path update for package installation
+
+On Windows, when a user installs a tool using package managers like `WinGet` and `Chocolatey` in PowerShell,
+the tool is often added to the system or user PATH environment variable. However, changes to PATH made during
+the installation are not automatically propagated to the running PowerShell session. As a result, users frequently
+need to start a new PowerShell session to recognize and use the newly installed command-line tools.
+
+This behavior can be confusing and disrupts the workflow, especially in automation scenarios or when users expect
+tools to be immediately available post-installation. Addressing this limitation would improve usability, align with
+user expectations, and reduce friction in both interactive and scripted environments.
+
+## Motivation
+
+As a PowerShell user, I would like tools installed by package managers like `WinGet` or `Chocolatey` to be useable
+in my current session so that I can use the new tool without interrupting my existing work in the current session.
+
+## Solution in CMD
+
+CMD suffered with the same problem and the Windows Terminal team already [made a fix](https://microsoft.visualstudio.com/OS/_git/os.2020/pullrequest/10131240)
```

Comment:
It is not public.

Reply:
Yes, it's not public. The link is put there for those who have access to that internal pull request. It doesn't affect the completeness of the RFC because the fix is described in words below.